community/pdns/README.alpine

```diff
@@ -0,0 +1,6 @@
+When upgrading from 4.1.10 and previous:
+
+This release contains a fix for CVE-2019-10203
+Upgrading is not enough you need to manually apply the schema change
+
+ALTER TABLE domains ALTER notified_serial TYPE bigint USING CASE WHEN notified_serial >= 0 THEN notified_serial::bigint END;
```

Comment:
There may be a safe way to automate this. If we can do this step for the user, we probably should.